tests/e2e/tutorials/tutorialBase.js

```diff
@@ -675,7 +675,7 @@ class TutorialBase {
     await this.waitAndClick('mode-button-postro', s4lIframe);
     await this.takeScreenshot("Postpro");
     const algorithmTrees = await utils.getChildrenElementsBySelector(s4lIframe, '[osparc-test-id="tree-algorithm');
-    if (algorithmTrees.length !== 1) {
+    if (algorithmTrees.length < 1) {
```

Comment:
so if ``>1 `` there i no need to through anymore?

Reply:
The PostPro step must have the smash file preloaded. We used to have only one pipeline in the algorithm tree (TI Field + Surface viewer) and one week ago we added a second pipeline (HF Field + Surface viewer), that's why the test started failing. With this change we make sure that there is at least one pipeline in the algorithm tree.